src/app/utils/embeddings.ts

```diff
@@ -1,23 +1,29 @@
-
-import { OpenAIApi, Configuration } from "openai-edge";
+import { OpenAIApi, Configuration } from 'openai-edge';

```

Comment:
prettier seems more useful than editorconfig, since it formatted this file correctly